notebooks/Grid Search for Binary Classification.ipynb

```diff
@@ -60,7 +60,7 @@
     "    Y = np.concatenate((Y_a0, Y_a1), axis=None)\n",
     "\n",
     "    X = pd.DataFrame({\"credit_score_feature\": score_feature,\n",
-    "                      \"aux_data_feature\": A})\n",
+    "                      \"sensitive_features\": A})\n",
```

Comment:
I don't think that the name of the column in `X` should be the same as the keyword argument. I made the previous version `aux_data_feature` for this reason. Otherwise, we might give the impression that it's a 'magic' column name.

Reply:
good idea. replacing by "example_sensitive_feature". reasonable?

Reply:
That works.